esmf_regrid/experimental/unstructured_scheme.py

```diff
@@ -117,10 +108,12 @@ def _regrid_unstructured_to_rectilinear__prepare(src_mesh_cube, target_grid_cube
 
     # TODO: Record appropriate dimensions (i.e. which dimension the mesh belongs to)
 
-    grid_x, grid_y = get_xy_coords(target_grid_cube)
+    grid_x, grid_y = get_xy_dim_coords(target_grid_cube)
+    mesh = src_mesh_cube.mesh
+    assert mesh is not None
```

Comment:
Is this to be properly handled / exception raised in the future?  If so maybe a TODO added?